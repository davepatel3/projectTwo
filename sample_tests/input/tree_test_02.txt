float
allocate
25.0382
0
allocate
-23.6489
1
set_parent
2
0
allocate
-50.9536
0
ref_node
3
remove
2
set_parent
1
3
set_parent
1
0
remove
3
ref_node
1
allocate
-78.5999
1
remove
2
remove
1
allocate
-41.7901
0
peek_nodes
remove
3
allocate
-26.9298
0
allocate
-90.8169
2
remove
2
allocate
4.9174
0
ref_node
3
ref_node
3
allocate
-44.227
3
allocate
93.1162
3
ref_node
2
allocate
-36.3599
1
peek_nodes
ref_node
3
set_parent
2
0
allocate
35.8011
3
set_parent
4
2
peek_nodes
peek_nodes
remove
2
allocate
-43.3266
5
allocate
28.9438
1
allocate
26.283
1
peek_nodes
allocate
-42.7706
4
set_parent
1
5
allocate
-26.2827
3
remove
1
allocate
-81.383
4
set_parent
7
0
set_parent
7
4
remove
5
allocate
31.3191
0
allocate
-30.1509
3
allocate
89.8539
6
ref_node
3
allocate
77.5189
3
allocate
52.0158
1
allocate
4.77149
0
set_parent
2
5
set_parent
5
8
allocate
84.0149
5
peek_nodes
set_parent
1
7
peek_nodes
allocate
86.6714
1
ref_node
1
ref_node
1
set_parent
1
2
ref_node
10
allocate
-84.8236
9
remove
10
allocate
82.6283
9
quit
