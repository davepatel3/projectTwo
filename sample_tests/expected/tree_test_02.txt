allocate
1
allocate
2
set_parent
allocate
3
ref_node
-50.9536
parent: 0
children: 
remove
set_parent
set_parent
remove
ref_node
25.0382
parent: 0
children: 
allocate
2
remove
remove
allocate
3
peek_nodes
0: 0
1: [recycled]
2: [recycled]
3: -41.7901
remove
allocate
2
allocate
1
remove
allocate
3
ref_node
4.9174
parent: 0
children: 
ref_node
4.9174
parent: 0
children: 
allocate
1
allocate
2
ref_node
93.1162
parent: 3
children: 
allocate
4
peek_nodes
0: 0
1: -44.227
2: 93.1162
3: 4.9174
4: -36.3599
ref_node
4.9174
parent: 0
children: 1 2 
set_parent
allocate
5
set_parent
peek_nodes
0: 0
1: -44.227
2: 93.1162
3: 4.9174
4: -36.3599
5: 35.8011
peek_nodes
0: 0
1: -44.227
2: 93.1162
3: 4.9174
4: -36.3599
5: 35.8011
remove
allocate
4
allocate
2
allocate
6
peek_nodes
0: 0
1: -44.227
2: 28.9438
3: 4.9174
4: -43.3266
5: 35.8011
6: 26.283
allocate
7
set_parent
allocate
8
remove
allocate
2
set_parent
set_parent
remove
allocate
6
allocate
1
allocate
2
ref_node
4.9174
parent: 0
children: 8 1 
allocate
7
allocate
4
allocate
5
set_parent
set_parent
allocate
9
peek_nodes
0: 0
1: -30.1509
2: 89.8539
3: 4.9174
4: 52.0158
5: 4.77149
6: 31.3191
7: 77.5189
8: -26.2827
9: 84.0149
set_parent
peek_nodes
0: 0
1: -30.1509
2: 89.8539
3: 4.9174
4: 52.0158
5: 4.77149
6: 31.3191
7: 77.5189
8: -26.2827
9: 84.0149
allocate
10
ref_node
-30.1509
parent: 7
children: 4 10 
ref_node
-30.1509
parent: 7
children: 4 10 
set_parent
ref_node
86.6714
parent: 1
children: 
allocate
11
remove
allocate
10
quit
